source/_docs/load-and-performance-testing.md

```diff
@@ -4,197 +4,80 @@ description: Learn how to monitor internal execution performance of your Pantheo
+
+<div class="alert alert-info" role="alert">
+<h3 class="info">Note</h3>
+<p><strong>Performance testing environments should be uncached</strong>. Where possible, try to ensure that no-cache headers are applied to the response.
+</p></div>
+See [Bypassing Cache with HTTP Headers](/docs/cache-control)
+
+## How to Load/Performance Test your Site
+How to Load/Performance Test your Site
+The procedure for running a load test and a performance test are similar.
+
+###Setup New Relic
+First, [enable new relic pro](/docs/new-relic) to ensure that you have clear reporting on specific bottlenecks.  Set your <a target ="_blank" href="https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction#score">apdex</a> threshold according to your business rules (.5 is the default).  Be careful not to set this too high otherwise you will not get as many transaction traces in New Relic.  If you have any transactions that you want to ensure transaction trace is on be sure to set them up as <a target="_blank" href="https://docs.newrelic.com/docs/apm/transactions/key-transactions/key-transactions-tracking-important-transactions-or-events">key transactions</a>.
+
+###Use a load testing tool
+There are many tools that will accomplish both goals of load and performance testing.  Tools like:
+
+* SaaS Solutions
+ * <a target="_blank" href="https://www.blazemeter.com/">Blazemeter</a>
+ * <a target="_blank" href="https://loadimpact.com/">Load Impact</a>
+* Open Source tools
+ * <a target="_blank" href="http://jmeter.apache.org/">Jmeter</a>
+ * <a target="_blank" href="http://locust.io/">Locust.io</a>
+
+The Pantheon onboarding team uses Locust.io for load testing.  The benefit of locust is that is open source, it is fairly easy to build out test scripts and it allows you to crawl the site instead of using predefined urls for the load test.  Crawling the site has the added benefit of hitting every page that is linked to anywhere on the site.  This means that a lot of edge case performance bottlenecks can be found where other more rigid tests would have not.
+
+Ultimately, is doesn't matter what tool you decide to use as long as it allows you to test your site properly.  Be sure to allow for any authenticated traffic as well as anonymous.  Which brings the next step, how to determine how much load to apply.  
+
+For performance tests a smaller load should suffice as you should be able to see transactional bottlenecks with 10-20 concurrent users.
+
+For load tests, to determine how many concurrent users you must start with looking at your historical analytics of your site.  Try to find the peak hourly sessions of your site and the average session duration. 

```

Comment:
@bentekwork how do I determine load to apply in the test after calculating concurrent users?